ledger/ledger-utxo/src/main/java/net/corda/v5/ledger/utxo/NotarySignatureVerificationService.java

```diff
@@ -0,0 +1,43 @@
+package net.corda.v5.ledger.utxo;
+
+import net.corda.v5.application.crypto.DigitalSignatureAndMetadata;
+import net.corda.v5.base.annotations.DoNotImplement;
+import net.corda.v5.crypto.SecureHash;
+import net.corda.v5.ledger.common.transaction.TransactionSignatureException;
+import org.jetbrains.annotations.NotNull;
+import java.security.PublicKey;
+import java.util.List;
+import java.util.Map;
+
+@DoNotImplement
+public interface NotarySignatureVerificationService {
+
+    /**
+     * Verifies given transaction by id is signed by the notary signatures.
+     *
+     * @param transactionId of signatures
```

Comment:
That line does not make a lot of sense - do you mean something like
```suggestion
     * @param transactionId that is exepcted to be signed over
```